Source/DafnyTestGeneration/DeadCodeCommand.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Linq;
+
+namespace Microsoft.Dafny; 
+
+public class DeadCodeCommand : ICommandSpec {
+  public IEnumerable<IOptionSpec> Options =>
+    new IOptionSpec[] {
+      LoopUnrollOption.Instance,
+      SequenceLengthLimitOption.Instance,
+      VerificationTimeLimitOption.Instance,
+    }.Concat(ICommandSpec.CommonOptions);
+
+  public Command Create() {
+    var result = new Command("find-dead-code", "(Experimental) Use block-coverage tests to warn about potential dead code.");
```

Comment:
Is this documented somewhere?

Reply:
It's documented quite well in https://github.com/dafny-lang/dafny/tree/master/Source/DafnyTestGeneration, but we don't mention it in the reference manual or user guide or anything like that.

Reply:
I'd reword the description of the feature slightly though, because it doesn't actually involve block-coverage tests, it just uses the same common technique of trying to find a counterexample for an `assert false;`. How about:

```suggestion
    var result = new Command("find-dead-code", "(Experimental) Use counterexample generation to warn about potential dead code.");
```